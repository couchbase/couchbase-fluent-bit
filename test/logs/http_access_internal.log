127.0.0.1 - @ [09/Mar/2021:17:32:02 +0000] "RPCCONNECT /goxdcr-cbauth HTTP/1.1" 200 0 - Go-http-client/1.1
::1 - Administrator [09/Mar/2021:17:32:13 +0000] "GET /pools/default HTTP/1.1" 404 14 - Go-http-client/1.1
10.244.1.2 - Administrator [09/Mar/2021:17:32:14 +0000] "GET /pools HTTP/1.1" 200 541 - couchbase-operator/2.2.0 (commit/10459d7c9b0642eb398bdc59ad45b26201065ba1; build/999)
127.0.0.1 - @ [09/Mar/2021:17:32:14 +0000] "RPCCONNECT /index-cbauth HTTP/1.1" 200 0 - Go-http-client/1.1
127.0.0.1 - @index-cbauth [09/Mar/2021:17:32:14 +0000] "GET /_metakv/indexing/settings/config HTTP/1.1" 200 937 - Go-http-client/1.1
