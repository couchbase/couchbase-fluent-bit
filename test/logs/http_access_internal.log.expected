/fluent-bit/test/logs/http_access_internal.log.actual: [1615311122.000000000, {"file_path":"/fluent-bit/test/logs","file":"http_access_internal.log","filename":"/fluent-bit/test/logs/http_access_internal.log","host":"127.0.0.1","user":"@","timestamp":"09/Mar/2021:17:32:02 +0000","method":"RPCCONNECT","path":"/goxdcr-cbauth","code":"200","size":"0","client":"Go-http-client/1.1","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/http_access_internal.log.actual: [1615311133.000000000, {"file_path":"/fluent-bit/test/logs","file":"http_access_internal.log","filename":"/fluent-bit/test/logs/http_access_internal.log","host":"::1","user":"Administrator","timestamp":"09/Mar/2021:17:32:13 +0000","method":"GET","path":"/pools/default","code":"404","size":"14","client":"Go-http-client/1.1","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/http_access_internal.log.actual: [1615311134.000000000, {"file_path":"/fluent-bit/test/logs","file":"http_access_internal.log","filename":"/fluent-bit/test/logs/http_access_internal.log","host":"10.244.1.2","user":"Administrator","timestamp":"09/Mar/2021:17:32:14 +0000","method":"GET","path":"/pools","code":"200","size":"541","client":"couchbase-operator/2.2.0 (commit/10459d7c9b0642eb398bdc59ad45b26201065ba1; build/999)","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/http_access_internal.log.actual: [1615311134.000000000, {"file_path":"/fluent-bit/test/logs","file":"http_access_internal.log","filename":"/fluent-bit/test/logs/http_access_internal.log","host":"127.0.0.1","user":"@","timestamp":"09/Mar/2021:17:32:14 +0000","method":"RPCCONNECT","path":"/index-cbauth","code":"200","size":"0","client":"Go-http-client/1.1","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO","level_numeric":"20"}]
/fluent-bit/test/logs/http_access_internal.log.actual: [1615311134.000000000, {"file_path":"/fluent-bit/test/logs","file":"http_access_internal.log","filename":"/fluent-bit/test/logs/http_access_internal.log","host":"127.0.0.1","user":"@index-cbauth","timestamp":"09/Mar/2021:17:32:14 +0000","method":"GET","path":"/_metakv/indexing/settings/config","code":"200","size":"937","client":"Go-http-client/1.1","hostname":"test-container","logshipper":"couchbase.sidecar.fluentbit","fluentbit.version":"99","pod":{"name":"test-pod"},"couchbase":{"logging.version":"99","cluster":"test-cluster"},"level":"INFO","level_numeric":"20"}]
